project/Build.scala

```diff
@@ -0,0 +1,49 @@
+import com.typesafe.sbt.{GitPlugin, SbtScalariform}
+import de.heikoseeberger.sbtheader.HeaderPlugin
+import de.heikoseeberger.sbtheader.license.Apache2_0
+import sbt._
+import sbt.plugins.JvmPlugin
+import sbt.Keys._
+import sbtassembly.AssemblyPlugin
+
+import scalariform.formatter.preferences.{AlignSingleLineCaseStatements, DoubleIndentClassDeclaration}
+
+object Build extends AutoPlugin {
+
+  override def requires = JvmPlugin && HeaderPlugin && GitPlugin && SbtScalariform
+
+  override def trigger = allRequirements
+
+  override def projectSettings = Vector(
+    // Core settings
+    organization := "de.bripkens",
+    licenses += ("Apache-2.0", url("http://www.apache.org/licenses/LICENSE-2.0")),
```

Comment:
Oh sorry, I did't take enough care. It's your choice which license to use. Of corse I favor ALv2 over MIT :-) The headers plugin can also generate a MIT license header.